project.clj

```diff
@@ -32,8 +32,6 @@
                       :dependencies [[org.clojure/clojure "1.11.0-master-SNAPSHOT"]
                                      [org.clojure/clojurescript "1.10.439"]]}
 
-             :nrepl-0.4 {:dependencies [[nrepl/nrepl "0.4.5"]]}
```

Comment:
We'll have to mention in the README/Changelog that Piggieback now requires nREPL 0.6+.

Reply:
> We'll have to mention in the README/Changelog that Piggieback now requires nREPL 0.6+.

Updated the README & CHANGELOG with new nREPL version requirement changes.